src/tests/mockData/profile.js

```diff
@@ -0,0 +1,43 @@
+const profile = {
+  user: {
+    username: 'seremnn',
+    email: 'serem@testing.com',
+    token: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6OCwidXNlcm5hbWUiOiJzZXJlbW5uIiwiZXhwIjoxNTU1Mjc0OTgwfQ.DVRlpBQWpgt5mLIwumzgeOS8eHXCMpSHk6XpJaqMPn0',
+    profile: {
+      bio: 'ss',
+      city: 'mkjbhvg',
+      country: '',
+      first_name: null,
+      following: false,
+      image: 'image/upload/v1552193974/gyzbaptikqalgthxfdnh.png',
+      image_url: 'https://res.cloudinary.com/dbsri2qtr/image/upload/c_fill,h_150,w_100/gyzbaptikqalgthxfdnh',
+      last_name: null,
+      phone: 0,
+      website: '',
+      username: 'seremnn',
+    },
+  },
+};
+
+export default profile;
```

Comment:
@BrianSerem, do not make default exports in this file.

Reply:
I have made this a named import